tests/runner.js

```diff
@@ -1,8 +1,5 @@
 'use strict';
 
-const captureExit = require('capture-exit');
```

Comment:
I believe we can remove this as far as now we don't use real `process.exit()` in unit tests.